src/interfaces/Whip_DismissStorage.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Interface Whip_DismissStorage
+ */
+interface Whip_DismissStorage {
+
+	/**
+	 * Saves the value.
+	 *
+	 * @param string $dismissedVersion The value to save.
```

Comment:
Also add a return value with type boolean, to be able to confirm save has been successful.